src/scrape_up/github/users.py

```diff
@@ -25,6 +25,18 @@ def followers(self):
             message = f"{self.username} not found !"
             return message
 
+    def following(self):
+        """"
```

Comment:
Include this method in `documentation.md` file.

Reply:
@nikhil25803 added the function